### PR TITLE
Translated project-administration\role-and-member-management.md.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>
/assign @Sherlock113

### CODE REVIEW

File: content/zh/docs/project-administration/role-and-member-management.md [hunk 1]
```diff
@@ -1,92 +1,92 @@
...
+| admin     | 项目管理员，可以对项目下的所有资源执行所有操作。此角色可以完全控制项目下的所有资源。 |
 
-1. In **Project Roles**, click `admin` and you can see the role detail as shown below.
+1. 在**项目角色**页面，点击 `admin` 查看该角色的详情，如下图所示：
 
-    ![view role details](/images/docs/project-admin/project_role_detail.png)
+    ![view role details](/images/docs/zh-cn/project-admin/project_role_detail.png)
 
-2. You can switch to **Authorized Users** tab to see all the users that are granted an `admin` role.
+2. 您可以点击**授权用户**页签查看被授予 `admin` 角色的用户。
```
Comment (Sherlock113) [inline content/zh/docs/project-administration/role-and-member-management.md:39]: 页签 -> 选项卡

**Review by Sherlock113 — COMMENTED** [review]
Make sure the role you use in the CN image matches the description in the guide. Please refer to the English version. Do not use the `admin` account for all the operations.

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on on comments.
/assign @Sherlock113

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on on comments.
/assign @Sherlock113

**Comment by Patrick-LuoYu** [comment]
Fixed issues based on on comments.
/assign @Sherlock113

**Review by rayzhou2017 — APPROVED** [review]

**Review by rayzhou2017 — APPROVED** [review]

**Pull request merged**